packages/expo-contacts/android/src/main/java/expo/modules/contacts/ContactsModule.kt

```diff
@@ -281,8 +263,20 @@ class ContactsModule : Module() {
         return@AsyncFunction
       }
       // Create contact from supplied data.
-      val contact = mutateContact(null, contactData)
-      presentForm(contact)
+      if (contactData != null) {
+        val contact = mutateContact(null, contactData)
+        mPendingPromise = promise
+        presentForm(contact)
+      }
+      promise.resolve()
+    }
+
+    OnActivityResult { _, payload ->
```

Comment:
@alanjhughes This seems to be a new API in the module definition? Could this be documented in the official docs?